lib/ssh/src/ssh_client_channel.erl

```diff
@@ -205,9 +205,13 @@ The following message is taken care of by the `ssh_client_channel` behavior.
 %%====================================================================
 %% API
 %%====================================================================
-
 -doc(#{equiv => call/3}).
 -doc(#{since => <<"OTP 21.0">>}).
+-spec call(ChannelRef, Msg) -> Reply | {error, Reason} when
+      ChannelRef :: pid(),
```

Comment:
Todays documentation makes channelRef opaque, this will make it a pid!

Reply:
I've reread the PR and still can't understand what you meant. Please let me know what you meant.